src/backend/app/services/dataset_service.py

```
@@ -36,82 +105,104 @@ async def process_dataset_upload(self, file: UploadFile) -> dict:
+    async def _prepare_company_investor_relations(self, rows: list[dict], company_name_to_id: dict) -> list[dict]:
         relations = []
         for row in rows:
             company_id = company_name_to_id.get(row.get('Company Name', '').strip())
             if not company_id:
                 continue
             if 'Top Investors' in row and row['Top Investors']:
                 for inv in row['Top Investors'].split(','):
-                    investor_name = inv.strip()
-                    investor_obj = await investor_repo.get_by_name(investor_name)
+                    original_investor_name = inv.strip()
+                    normalized_investor_name = self._normalize(original_investor_name)
```

Ruff (0.11.9)

Local variable `normalized_investor_name` is assigned to but never used

Remove assignment to unused variable `normalized_investor_name`

(F841)